docs/src/main/sphinx/release/release-401.md

```diff
@@ -5,23 +5,35 @@
 * Add support for using path-style access for all requests to S3 when using
   fault-tolerant execution with exchange spooling. This can be enabled with the
   `exchange.s3.path-style-access` configuration property. ({issue}`14655`)
+* Add support for polymorphic table functions in file-based access control. ({issue}`13713`)
 * Add distribution data for output buffer utilization to
   `EXPLAIN ANALYZE VERBOSE`. ({issue}`14596`)
 * Add distribution data for operator blocked time to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14640`)
 * Improve performance and reliability of `INSERT`, `MERGE`, and table optimization. ({issue}`14553`)
+* Fix query failure caused by a
+  `com.google.common.base.VerifyException: cannot unset noMoreSplits` error. ({issue`14668`})
+* Fix underestimation of CPU usage and scheduled time statistics for the build
+  side of joins in `EXPLAIN ANALYZE`. ({issue}`14572`)
 
 ## Delta Lake connector
 
 * Add support for writing to tables with [Delta Laker writer protocol version 4](https://docs.delta.io/latest/versioning.html#features-by-protocol-version).
   This does not yet include support for [change data feeds](https://docs.delta.io/2.0.0/delta-change-data-feed.html)
   or generated columns. ({issue}`14573 `)
+* Add support for writes on Google Cloud Storage. ({issue}`12264`)
+* Add support for DML statements on tables backed by GCS
```

Comment:
DML ?? might have to detail that.. also GCS .. maybe spell out again or add in line above

Reply:
The line above was the fix for it, I just forgot to delete. 😆